participants/eduardosand/pr_tutorial/simple_functions.py

```diff
@@ -6,7 +6,15 @@ def fibonacci(max):
 
 
 def factorial(value):
+    """
+    Return the factorial of a function
+    :param value: (float)
```

Comment:
Factorial (though not as written here) is not defined for floating point numbers. Maybe change to integer?